xml/ha_config_basics.xml

```diff
@@ -447,16 +447,10 @@
        supports it. For more information, refer to
        <xref linkend="sec.ha.config.basics.resources.advanced.clones"/>.
       </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term>Multi-state Resources (formerly known as Master/Slave Resources)</term>
-     <listitem>
       <para>
-       Multi-state resources are a special type of clone resources that can
-       have multiple modes. For more information, refer to
-       <xref linkend="sec.ha.config.basics.resources.advanced.masters"/>.
-      </para>
+       Multi-state resources (formerly known as master/slave resources) are a
```

Comment:
Or  "Promotable resources (formaerly known as master/slave reosurces, or multi-state resources)"?

Reply:
Used 'promotable clones' (same term that is used the upstream docs).